CHANGELOG.md

```diff
@@ -1,6 +1,16 @@
 # OHHTTPStubs — CHANGELOG
 
+## [Future Release]
+
+* Updating default Swift Version to 4.2, but keeping code compiling in Swift 2.3, 3.x, and 4.x.  
```

Comment:
Would be interesting to mention that only 4.x+ is tested on CI so 2.3 compatibility for example isn't guaranteed by tests and might break at any time because we don't test it anymore.

(Might actually be simpler to just drop the mention of 2.x altogether anyway and stop mentioning it tbh)

Reply:
Updated in c9b28c9.  I went with the path of removing 2.x mentions.